weblab/experiments/views.py

```diff
@@ -93,9 +106,13 @@ def get(self, request, *args, **kwargs):
         # Only give info on experiments involving the correct entity versions
         experiments = {}
         for exp in Experiment.objects.filter(model__in=q_models, protocol__in=q_protocols):
-            if (exp.model_version == models[exp.model.pk]['version'] and
-                    exp.protocol_version == protocols[exp.protocol.pk]['version']):
-                experiments[exp.pk] = self.experiment_json(exp)
+            if ((exp.model_version == models[exp.model.pk]['version'] and
```

Comment:
Why the double (())?

Reply:
pep8-friendly indentation fix to prevent confusing alignment in multiline if ... although I'm not actually sure why I changed this, since the original code handles this, just in a different way (extra indent level for second part of clause) :smile: